src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --header-fs: 12px;
+  --subheader-fs: 10px;
+  --link-color: #00acdc;
+  --link-height: 40px;
+  --link-fs: 14px;
```

Comment:
in scss you can define variable like this: 

```scss
  $mainColor: #060b35;
  $secondaryColor: #616070;
  $headerFs: 12px;
```

read more about variables in sass - [link](https://www.w3schools.com/sass/sass_variables.php)